spec/features/budget/new.html.erb_spec.rb

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
- [ ] There are some test specs **failing**. Kindly ensure to improve upon those aspects.

**Failing rspec tests:**
![image](https://user-images.githubusercontent.com/12745474/144693487-61786d7e-57ff-4891-ad46-503d1ef6ba3a.png)

Reply:
![image](https://user-images.githubusercontent.com/54015740/144711149-f725faba-c670-4683-be73-420be1c92cdf.png)

Reply:
corrected